docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,885 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #212.                _foreward_process(T, m, M_T, Σ_T, T_subseq_isconstant, excl_zone, i, bsf, pruned)
</pre><p>Instead of <code>_forward_process</code>, we may say: <code>_update_pruned</code>   </p><p> </p><p>And, similarly, instead of <code>_backward_process</code> , we may say: <code>_calculate_distance</code> . As a follow up to my previous comment, we can just return <code>nn_distance</code> . In line #213 (i.e. next line after the line above), we can have:</p><p> </p><pre class="ql-syntax" spellcheck="false">if nn_distance &gt; bfs:  
    bfs = nn_distance  # update discord score

# alternatively: bfs = max(bfs, nn_distance)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>